test/internet/test-dns.js

```diff
@@ -553,7 +553,7 @@ console.log('looking up nodejs.org...');
 
 const cares = process.binding('cares_wrap');
 const req = new cares.GetAddrInfoReqWrap();
-cares.getaddrinfo(req, 'nodejs.org', 4);
+cares.getaddrinfo(req, 'nodejs.org', 4, /* hints */ 0, /* verbatim */ true);
```

Comment:
That assumes python's implementation is exactly the same as node's, though.  If it's not, or worse, not always, it's going to be a right pain in the neck to debug.

If you still think it's a good idea, can you file a separate issue to hash out the details?